conans/client/conf/__init__.py

```diff
@@ -43,7 +43,9 @@
     SunOS:
     Arduino:
         board: ANY
-arch: [x86, x86_64, ppc32, ppc64le, ppc64, armv6, armv7, armv7hf, armv7s, armv7k, armv8, armv8_32, armv8.3, sparc, sparcv9, mips, mips64, avr]
+    Neutrino:
+        version: ["6.4.0", "6.5.0", "7.0.0"]
```

Comment:
Are compiled binaries incompatible with different versions of the OS?

Reply:
no idea, need to investigate it further

Reply:
it seems to be forward-compatible (but not backward compatible)
http://www.qnx.com/developers/docs/6.5.0/index.jsp?topic=%2Fcom.qnx.doc.ide.userguide%2Ftopic%2Fmigrate_Binary_Compatibility.html